cli/src/main/kotlin/commands/AnalyzerCommand.kt

```diff
@@ -144,13 +151,21 @@ class AnalyzerCommand : CliktCommand(name = "analyze", help = "Determine depende
 
         println("Analyzing project path:\n\t$inputDir")
 
-        val analyzerConfig = AnalyzerConfiguration(ignoreToolVersions, allowDynamicVersions)
+        val config = globalOptionsForSubcommands.config
+        val analyzerConfig = AnalyzerConfiguration(
+            ignoreToolVersions,
+            allowDynamicVersions,
+            config.analyzer?.sw360Configuration
+        )
         val analyzer = Analyzer(analyzerConfig)
 
         val curationProvider = FallbackPackageCurationProvider(
             listOfNotNull(
                 packageCurationsFile.takeIf { it.isFile }?.let { FilePackageCurationProvider(it) },
-                ClearlyDefinedPackageCurationProvider().takeIf { useClearlyDefinedCurations }
+                ClearlyDefinedPackageCurationProvider().takeIf { useClearlyDefinedCurations },
+                config.analyzer?.sw360Configuration?.let {
+                    Sw360PackageCurationProvider(config.analyzer?.sw360Configuration).takeIf { useSw360Curations }
+                }
```

Comment:
The order in this list defines the priority of the providers, I would assume that the SW360 provider should be above the ClearlyDefined provider.